.travis.yml

```diff
@@ -21,10 +21,18 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-19-a
     - os: osx
       osx_image: xcode9.2
```

Comment:
Can you change this to Xcode9.4? That version includes Swift 4.1.2 and means there will be no Swift binary download for that job